Breeze.UI/src/app/login/login.component.ts

```diff
@@ -45,39 +59,36 @@ export class LoginComponent implements OnInit {
     if (!this.openWalletForm) { return; }
     const form = this.openWalletForm;
     for (const field in this.formErrors) {
+      if (!this.formErrors.hasOwnProperty(field)) {
```

Comment:
this can be done with a positive statement w/o continue around the whole loop. It's easier to read
```
if (this.formErrors.hasownProperty(field)) {
...
}
```

Reply:
This is mainly from resharper suggestion, I also prefer it this way as it makes code less nested